src/components/ListItemIcon/ListItemIcon.vue

```diff
@@ -174,6 +174,14 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+
+		/**
+		 * Unique list item ID
+		 */
+		id: {
+			type: String,
+			default: '',
```

Comment:
In case there is no id given to the component, does that work then?
Afaik on html, the id-prop is not allowed to be empty if present, no? So while it would probably work on most browsers, it won't produce valid html...? 🤔 Or does vue just throw it away, if it's an empty string?

Reply:
Good catch. There are now a lot of `id=""` in the HTML code.

Reply:
ah right, should be null!

Reply:
Done!